Makefile

```diff
@@ -262,6 +262,21 @@ update/gomod:
 update/shfmt: bin/shfmt
 	./bin/shfmt -w -i 2 -d ./hack/
 
+.PHONY: update/add-license-header
+update/add-license-header:
```

Comment:
Consider hiding this as a a `/hack` shell script as to not pollute the Makefile as we do for other scripts like `./hack/release-scripts/generate-sidecar-tags` and `./hack/update-kustomize.sh`.